contracts/multisig/src/key.rs

```diff
@@ -339,12 +344,24 @@ mod ecdsa_tests {
         let hex = HexBinary::from_hex("049b").unwrap();
         assert_eq!(
             PublicKey::try_from((KeyType::Ecdsa, hex.clone())).unwrap_err(),
-            ContractError::InvalidPublicKeyFormat {
-                reason: "Invalid input length".into()
-            }
+            ContractError::InvalidPublicKey
         );
     }
 
+    #[test]
+    fn should_normalize_compressed_and_uncompressed_public_keys() {
+        let uncompressed = HexBinary::from_hex("049bb8e80670371f45508b5f8f59946a7c4dea4b3a23a036cf24c1f40993f4a1daad1716de8bd664ecb4596648d722a4685293de208c1d2da9361b9cba74c3d1ec").unwrap();
+        let compressed = HexBinary::from_hex(
+            "029bb8e80670371f45508b5f8f59946a7c4dea4b3a23a036cf24c1f40993f4a1da",
+        )
+        .unwrap();
+
+        assert_eq!(
+            PublicKey::try_from((KeyType::Ecdsa, uncompressed.clone())),
+            PublicKey::try_from((KeyType::Ecdsa, compressed.clone()))
```

Comment:
```suggestion
            PublicKey::try_from((KeyType::Ecdsa, uncompressed.clone())).unwrap(),
            PublicKey::try_from((KeyType::Ecdsa, compressed.clone())).unwrap()
```
If both `try_from` return `Err` instead of `Ok`, the test would still pass but that's no good.